pkg/stage0

```diff
@@ -1,9 +1,10 @@
 [deps]
+jobflow
```

Comment:
if we build jobflow in stage0, we must use musl-gcc like it was used in the butch recipe and put the right deps